boefjes/boefjes/job_handler.py

```diff
@@ -131,22 +132,35 @@ def handle(self, boefje_meta: BoefjeMeta) -> None:
 
             boefje_meta.arguments["input"] = ooi.serialize()
 
+        boefje_results: list[tuple[set, bytes | str]] = []
+
         boefje_meta.runnable_hash = plugin.runnable_hash
-        boefje_meta.environment = get_environment_settings(boefje_meta, plugin.boefje_schema)
 
         logger.info("Starting boefje %s[%s]", boefje_meta.boefje.id, str(boefje_meta.id))
 
         boefje_meta.started_at = datetime.now(timezone.utc)
```

Comment:
We should probably move this line a bit lower, after the get_envinrment_settings, as fetching and handling those is not actual part of the runtime of the boefje itself.

Reply:
Good suggestion, should be fixed now